src/aks-preview/HISTORY.md

```diff
@@ -13,6 +13,7 @@ Release History
 
 0.5.26
 +++++
+* Add podMaxPids argument for node config
```

Comment:
It is not appropriate to add history in this way, since 0.5.26 will not be released again to include these code changes.

Reply:
Hey @FumingZhang apologies, do I just leave History.md alone then ?